packages/deployer/src/commands.ts

```diff
@@ -16,9 +16,63 @@ import {
   generateMnemonic,
   deriveProviderId,
   shortenProviderId,
-  waitForEnter,
   deriveMasterWalletAddress,
 } from './util';
+import readline from 'readline';
+
+function shuffleArray(array) {
```

Comment:
```suggestion
// Adapted from https://stackoverflow.com/a/12646864/14558682
function shuffleArray(array) {
```

Reply:
Lodash (which we already include) has a function for shuffling arrays:

```ts
import shuffle from 'lodash/shuffle';
```